hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/SQLFederationStateStore.java

```diff
@@ -1016,30 +1035,275 @@ private static byte[] getByteArray(ByteBuffer bb) {
   @Override
   public AddReservationHomeSubClusterResponse addReservationHomeSubCluster(
```

Comment:
Is there a way to add tests to any of this?

Reply:
Thank you for your suggestion.

I will explain this part of the test:
We are in YARN-11272, we implemented store Reservations in ZK, in this pr we defined the test of the method in FederationStateStoreBaseTest.java

Reply:
There's nothing we need to enable the test then?